src/utils/constants.js

```diff
@@ -5,7 +5,7 @@ export const OBJECT_TYPE_ARROW = 'Arrow';
 export const OBJECT_TYPE_AXES = 'Axes';
 export const OBJECT_TYPE_FLAT_ARROW = 'FlatArrow';
 
-export const MAX_POINTCLOUD_POINTS = 100000;
+export const MAX_POINTCLOUD_POINTS = 500000;
```

Comment:
Many pointclouds do not fit inside 100,000 points. 500,000 seems to be a better default to avoid confusion to the users (comes at the cost of extra memory). Ideally the buffer should resize if the number of data points change.